File: app/src/main/java/com/rtukpe/fixtures/ui/home/DashboardPresenter.java
package com.rtukpe.fixtures.ui.home;

import com.rtukpe.fixtures.data.manager.DataManager;
import com.rtukpe.fixtures.ui.base.BasePresenter;
import com.rtukpe.fixtures.utils.rx.SchedulerProvider;

import javax.inject.Inject;

import io.reactivex.disposables.CompositeDisposable;

/**
 * Created by rtukpe on 21/03/2018.
 */

public class DashboardPresenter<V extends DashboardMvpView> extends BasePresenter<V> implements DashboardMvpContract<V> {

    @Inject
    public DashboardPresenter(DataManager dataManager, SchedulerProvider schedulerProvider, CompositeDisposable compositeDisposable) {
        super(dataManager, schedulerProvider, compositeDisposable);
    }

    @Override
    public void onViewInitialized() {
        super.onViewInitialized();
    }
}

File: app/src/main/java/com/rtukpe/fixtures/ui/home/DashboardPresenter.kt
package com.rtukpe.fixtures.ui.home

import com.rtukpe.fixtures.data.manager.DataManager
import com.rtukpe.fixtures.ui.base.BasePresenter
import com.rtukpe.fixtures.utils.rx.SchedulerProvider
import io.reactivex.disposables.CompositeDisposable
import javax.inject.Inject

/**
 * Created by rtukpe on 21/03/2018.
 */

class DashboardPresenter<V : DashboardMvpView>

@Inject
constructor(dataManager: DataManager, schedulerProvider: SchedulerProvider, compositeDisposable: CompositeDisposable) :
        BasePresenter<V>(dataManager, schedulerProvider, compositeDisposable), DashboardMvpContract<V>
